pages/common/dhclient.md

```diff
@@ -0,0 +1,11 @@
+# dhclient
+
+> DHCP client.
+
+- Get an IP address for the eth0 interface:
```

Comment:
Put eth0 within backticks(``)

Reply:
Done in 7d0f7d37b298915bf4cafe2f7fc4d4732556f20d